SmoothStackProject1/src/driver/Main.java

```diff
@@ -0,0 +1,148 @@
+package driver;
+
+import java.util.Scanner;
+
+import services.AuthorService;
+import services.BookService;
+import services.PublisherService;
+
+public class Main {
+
+	static Scanner scan = new Scanner(System.in);
+	private static int selection;
+	
+	public static void menu() {
```

Comment:
I like the way you divided up the menu. That is a good idea. So you do not just have a huge nested if else or switch statement!